app/controllers/legal_docs_controller.rb

```diff
@@ -0,0 +1,44 @@
+class LegalDocsController < ApplicationController
+  before_filter :find_legal_doc, except: [:index, :new, :create]
```

Comment:
@browning25 this is where you need to include the SmartListing helpers, check out this line in the EmployeesController

then, in the index view, you can do:

```
def index
  legal_docs_scope = LegalDoc.all
  smart_listing_create :legal_docs,  legal_docs_scope,  partial: "legal_docs/listing"
end
```

OR -- you can skip using SmartListing entirely. the whole point of this index method is to identify _which_ legal doc to show, and then redirect to that legal doc.

SO, you could do something like this to accomplish that

```
def index
  redirect_to legal_doc_path(LegalDoc.where(type: params[:type], is_default: true).last)
end
```